content/movie-review-pt2.md

```diff
@@ -43,13 +46,9 @@ struct NoteState {
 
 ### Using Borsh for Serialization and Deserialization
 
-To access the data store on an account, we must serialize and deserialize the data. 
-1. Serialization is the process of converting an object into a bytes array (an area of memory containing a continuous sequence of bytes). 
-2. Deserialization is the process of reconstructing an object from a byte array. 
-
-For our use case, the object is simply the data we want to store. Writing to the data field of an account requires that we serialization the data we want to store. To read the data from an existing account, we must deserialized the account data from bytes into readable objects. We can then update this (deserialized) data and serialize the data into the account again. 
+Just as with instruction data, we need a mechanism for converting from our Rust data type to a byte array, and vis versa. **Serialization** is the process of converting an object into a byte array. **Deserialization** is the process of reconstructing an object from a byte array.
```

Comment:
- typo "vis versa" --> vice versa
- 2¢ "object to/from byte array" feels dry to me... which is good and clear, which adequate to our audience, but Ivan 'splained it to me as "human readable" <> byte Buffer.  sugg: add something to the effect of, "another way to think of it is converting to and from something human readable to a byte array."

Reply:
That's not technically correct though. It happens to be somewhat human readable but technically it's about converting back and forth between formats. Neither of the formats necessarily have to be human readable